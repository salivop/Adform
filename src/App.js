import React from "react";

import CampaignsTableContainer from "views/CampaignsTable/CampaignsTableContainer";

const App = () => {
  return (
    <React.Fragment>
      <CampaignsTableContainer />
    </React.Fragment>
  );
}

export default App;
