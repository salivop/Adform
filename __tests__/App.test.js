import React from "react";
import toJson from 'enzyme-to-json';
import { shallow } from "enzyme";

import App from "src/App";

describe("CampaignsTableContainer", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<App />)));

  it("should render correctly", () => {
    const component = wrapper;
    expect(toJson(component)).toMatchSnapshot();
  });

  it("should have Fragment", () => {
    expect(wrapper.find('Fragment').length).toEqual(1);
  })

  it("should have CampaignsTableContainer", () => {
    expect(wrapper.find('CampaignsTableContainer').length).toEqual(1);
  })
});
