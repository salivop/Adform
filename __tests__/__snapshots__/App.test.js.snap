// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampaignsTableContainer should render correctly 1`] = `
<Fragment>
  <CampaignsTableContainer />
</Fragment>
`;
